cupy/_logic/truth.py

```diff
@@ -255,6 +255,44 @@ def setdiff1d(ar1, ar2, assume_unique=False):
     return ar1[in1d(ar1, ar2, assume_unique=True, invert=True)]
 
 
+def setxor1d(ar1, ar2, assume_unique=False):
+    """Find the set exclusive-or of two arrays.
+
+    Parameters
+    ----------
+    ar1, ar2 : cupy.ndarray
+        Input arrays. They are flattend if they are not already 1-D.
+    assume_unique : bool
+        By default, False, i.e. input arrays are not unique.
+        If True, input arrays are assumed to be unique. This can
+        speed up the calculation.
+
+    Returns
+    -------
+    setxor1d : cupy.ndarray
+        Return the sorted, unique values that are in only one
+        (not both) of the input arrays.
+
+    See Also
+    --------
+    numpy.setxor1d
+
+    """
+    if not assume_unique:
+        ar1 = cupy.unique(ar1)
+        ar2 = cupy.unique(ar2)
+
+    aux = cupy.concatenate((ar1, ar2), axis=None)
+    if aux.size == 0:
+        return aux
+
+    aux.sort()
+    flag = aux
+    flag = cupy.concatenate((cupy.asanyarray([True]), cupy.asanyarray(
+        aux[1:] != aux[:-1]), cupy.asanyarray([True])), axis=None)
+    return aux[flag[1:] & flag[:-1]]
```

Comment:
By using `ElementwiseKernel`, we can compute `flag[1:] & flag[:-1]` without allocating additional memories for temporary values.

Reply:
@pri1311 How about this comment?

Reply:
Hey @asi1024! Sorry for the delay, was in between my exams, will get this done asap.